lib/cisco_node_utils/vpc.rb

```diff
@@ -170,11 +168,13 @@ def fabricpath_multicast_load_balance
       config_get('vpc', 'fabricpath_multicast_load_balance')
     end
 
-    def fabricpath_multicast_load_balance=(val)
-      set_args_keys(state: val ? '' : 'no')
-      # This requires fabricpath_emulated_switch_id to be set first
-      fail 'fabricpath_switch_id configuration is required in the vPC domain' if
-        !fabricpath_emulated_switch_id && val
+    def fabricpath_multicast_load_balance=(state)
+      FabricpathGlobal.fabricpath_enable
+
+      fail 'This property also requires fabricpath_emulated_switch_id' if
```

Comment:
Is this the case where the message generated by the switch is unclear?  If so, please add some comments to indicate why the specific fail message is needed.